web/modules/custom/portland/src/Plugin/Block/LoginBlock.php

```diff
@@ -31,12 +31,13 @@ public function getCacheMaxAge() {
      */
     public function build() {
       $logged_in = \Drupal::currentUser()->isAuthenticated();
-      $destination = \Drupal::request()->getRequestUri();
+      $destination = \Drupal::request()->query->get('destination');
```

Comment:
This doesn't seem to be working as intended. The editor login/logout link no longer contains a destination parameter.

Reply:
Updated the default destination.

Reply:
This doesn't really address the problem. Now the destination is incorrectly set to / instead of the actual page the user is on. For example, look at the link on https://pgov-1166-portlandor.pantheonsite.io/permits and compare it to the live site.

Reply:
I misunderstood what the original code was trying to do. Looks like it's trying to track which page the link is on. Should be fixed now.

Reply:
Yes, it's working correctly now. But I'm curious why did this code need to change in this PR? It was working before and the original code was more simple... did the old code not work with Entra for some reason?

Reply:
I've added some text in the Jira story. When clicked on repeated, the login link gets longer and longer as it copy and paste the whole URI every time. I thought it would be an easy fix so included it here.

Reply:
Ah, I see. Okay, makes sense now.